docs/docs-content/vm-management/vm-migration-assistant/create-migration-plans.md

```diff
@@ -508,13 +543,14 @@ Follow this guide to create migration plans using the VM Migration Assistant.
-   | **Plan name**        | A unique name for your migration plan.                                                                                                                                                                                                 | `myMigrationPlan`                         |
-   | **Target provider**  | Select the target provider from the drop-down Menu. By default, this will be your host cluster.                                                                                                                                        | `host`                                    |
-   | **Target namespace** | Select the target namespace for the VM migration from the drop-down Menu.                                                                                                                                                              | `myVmMigrationNamespace`                  |
-   | **Network map**      | A storage map defines the mapping of source storage domains to target storage classes or datastores, ensuring VM disks are correctly placed in the destination environment. Adjust the mapping, or leave the default mapping in place. | `VM Network` / `Pod Networking`           |
-   | **Storage map**      | A network map defines the mapping of source networks to target networks, ensuring VM network interfaces are correctly connected in the destination environment. Adjust the mapping, or leave the default mapping in place.             | `vsanDatastore` / `spectro-storage-class` |
+   | Setting                 | Description                                                                                                                                                                                                                            | Example                                   |
+   | ----------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------- |
+   | **Plan name**           | A unique name for your migration plan.                                                                                                                                                                                                 | `myMigrationPlan`                         |
+   | **Target provider**     | Select the target provider from the drop-down Menu. By default, this will be your host cluster.                                                                                                                                        | `host`                                    |
```

Comment:
What's a "host cluster"? We don't have such concept. Are you referring to the VMO cluster where the machines are being migrated to?